lightcycle-integration-test/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,28 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.soundcloud.lightcycle.sample.basic">
```

Comment:
what do you think about moving all this to a new package ? I think it would avoid the confusion with the samples. 
something like `com.soundcloud.lightcycle.integration_tests` or similar.